iati/guidance_and_support/models.py

```diff
@@ -61,6 +63,35 @@ class GuidancePage(AbstractContentPage):
     """A base for a single guidance page."""
     subpage_types = []
 
+    def get_context(self, request):
+        """Overwrite context to intercept POST requests to pages on this template and pass them to Zendesk API
+
+        Validate with some sort of captcha."""
+        context = super(GuidancePage, self).get_context(request)
+        form_success = "none"
+
+        if request.method == 'POST':
+            form_success = "failure"
```

Comment:
the use of a boolean (instead of a string) would feel a bit more pythonic, but not a big deal.